sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/client.py

```diff
@@ -164,8 +165,9 @@ def __init__(self, hostname, **kwargs):
         self._auth_timeout = kwargs.pop("auth_timeout", DEFAULT_AUTH_TIMEOUT)
         self._mgmt_links = {}
         self._retry_policy = kwargs.pop("retry_policy", RetryPolicy())
-        self._keep_alive_interval = int(kwargs.get("keep_alive_interval") or 0)
+        self._keep_alive_interval = int(kwargs.get("keep_alive_interval", 0))
         self._keep_alive_thread = None
+        self._lock = threading.Lock()
```

Comment:
Could we maybe name this something more descriptive?
Like `_mgmt_link_lock`, and move it up next to `self._mgmt_links={}` just for readability purposes :)